src/identity/mod.rs

```diff
@@ -71,3 +71,52 @@ struct BlsKeypairShare {
     pub secret: SerdeSecret<BlsSecretKeyShare>,
     pub public: BlsPublicKeyShare,
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use crate::ClientFullId;
+    use unwrap::unwrap;
+
+    #[test]
+    fn serialize_client_public_id_to_base64() {
```

Comment:
I think the three tests' names should include "encode" rather than "serialize".

Also, I think we should probably test failure paths of decoding too.